tools/accuracy_checker/accuracy_checker/adapters/__init__.py

```diff
@@ -47,6 +47,7 @@
     RetinaNetAdapter,
     ClassAgnosticDetectionAdapter,
     FaceBoxesAdapter,
+    PersonVehicleDetectionAdapter
```

Comment:
the reason of failure, missed comma
```suggestion
    PersonVehicleDetectionAdapter,
```